api/resources_portal/urls.py

```diff
@@ -89,3 +90,7 @@
     re_path(r"^$", RedirectView.as_view(url=reverse_lazy("api-root"), permanent=False)),
     path("v1/search/", include(search_router.urls)),
 ] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+
+urlpatterns.append(
+    path("v1/materials/import/", ImportViewSet.as_view({"post": "create"}), name="import")
```

Comment:
Any reason to not use `router.register` for this one?

Reply:
Only that I don't know how I would include multiple parameters in the URL if I did it with router.register. Do you know how to do that?

Reply:
You said in chat:

> I'm not sure how to register "material/import" without the system wanting it to be "material/<id>/import"

I agree that's a good reason. Can you just change the name to `materials-import` though?